assets/src/block-validation/sidebar.js

```diff
@@ -67,6 +75,7 @@ export function Sidebar() {
 	}, [] );
 
 	const saved = 'auto-draft' !== status;
+	const forceRevalidate = () => savePost( { preview: true } );
```

Comment:
This doesn't seem to be working properly. I published a post, modified some content, and then hit “Re-validate now”. Instead of creating an autosave revision, it actually updated the post entirely.

In other words, I was expecting a `POST` request to be made off to:

`https://wordpressdev.lndo.site/wp-json/wp/v2/posts/1987/autosaves?_locale=user`

But instead it made a `POST` request to:

`https://wordpressdev.lndo.site/wp-json/wp/v2/posts/1987?_locale=user`

Contrast this with clicking the “Preview in a new tab” which does the autosave request:

`https://wordpressdev.lndo.site/wp-json/wp/v2/posts/1987/autosaves?_locale=user`

Is there an autosaved dirty state that is separate from the non-autosaved dirty state? The “Re-validate now” button should be hidden when an autosave was made.

Reply:
I'll look into that 👍

Reply:
This has been addressed in d59b1eb